python/cudf/cudf/_version.py

```diff
@@ -14,6 +14,7 @@
 import re
 import subprocess
 import sys
+from typing import Any, Callable, Dict
```

Comment:
This file is generated by versioneer, can we ignore it?

Reply:
Done. The [recommended way](https://github.com/python/mypy/issues/4675) to do this seems to be to add the following to `setup.cfg`:

```
# [mypy-cudf._version]
# ignore_errors = True
```